pysrc/juliacall/__init__.py

```diff
@@ -50,6 +50,18 @@ def backtrace(self):
 
 def init():
     import os
+    if os.environ.get("CONDA_PREFIX", "") != "": #Only when calling from PythonCall
+        ## For Windows and Python 3.8+ we need to add dll search paths
+        import sys
+        import platform
+        if platform.system() == "Windows" and sys.hexversion >= 0x308000:
```

Comment:
```suggestion
        if platform.system() == "Windows" and sys.version_info >= (3, 8):
```
(Just for readability)